fineract-provider/src/main/java/org/apache/fineract/portfolio/common/BusinessEventNotificationConstants.java

```diff
@@ -25,47 +25,55 @@
-                                                                                                                                                                                                                                                                                "clients_reject"), CLIENTS_CREATE(
-                                                                                                                                                                                                                                                                                        "clients_create"), CENTERS_CREATE(
-                                                                                                                                                                                                                                                                                                "centers_create"), GROUPS_CREATE(
-                                                                                                                                                                                                                                                                                                        "groups_create"), SHARE_PRODUCT_DIVIDENDS_CREATE(
-                                                                                                                                                                                                                                                                                                                "share_product_dividends_create"), FIXED_DEPOSIT_ACCOUNT_CREATE(
-                                                                                                                                                                                                                                                                                                                        "fixed_deposit_account_create"), SHARE_ACCOUNT_APPROVE(
-                                                                                                                                                                                                                                                                                                                                "share_account_approve"), RECURRING_DEPOSIT_ACCOUNT_CREATE(
-                                                                                                                                                                                                                                                                                                                                        "recurring_deposit_account_create");
+        LOAN_APPROVED("loan_approved"), //
```

Comment:
@vincent-fuchs I would suggest to remove all these trailing `//` empty comments.. but it's too minor to nit pick on and hold back further reviewing and merging this, so I'll ignore it and move ahead. Perhaps you would like to raise a follow-up PR to remove these again, for it to be nicer?

Reply:
I could do it, but without these, spotless will complain and try to reformat it. Is there another way to tell spotless that it should not reformat ?

Reply:
Oh... I see. Didn't realize that. You could try to tweak [our Spotless config](https://github.com/apache/fineract/blob/develop/fineract-provider/build.gradle#L268), it's configured to use [our Eclipse Code Formatter style](https://github.com/apache/fineract/blob/develop/config/fineractdev-formatter.xml), which you could try to change, if this is important to you. FYI @thesmallstar is our resident expert in such matters!